packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -298,7 +293,9 @@ abstract class UnpackIOS extends Target {
       throw Exception('Binary $frameworkBinaryPath does not exist, cannot thin');
     }
     _thinFramework(environment, frameworkBinaryPath, archs);
-    _bitcodeStripFramework(environment, frameworkBinaryPath);
+    if (buildMode == BuildMode.release) {
```

Comment:
why this change?

Reply:
(Currently until we remove it) the release version of `Flutter.framework` has a giant bitcode blob, and now we always want to strip it out.

I deleted the logic passing the Xcode build setting `ENABLE_BITCODE` down into `assemble` as `kBitcodeFlag`.  It would only be true if `ENABLE_BITCODE` was set, AND it was an "install" command, meaning it was archiving.  
https://github.com/flutter/flutter/blob/b22db680089a2170e13a52f571bc14eec546c75e/packages/flutter_tools/bin/xcode_backend.dart#L336-L339

This was the check in the target that was deleted:
https://github.com/flutter/flutter/blob/723b82e4f0f3652dfaffb6e9bbc714e069da2976/packages/flutter_tools/lib/src/build_system/targets/ios.dart#L377-L380

Reply:
Ahh, makes sense. Thanks!